src/lib/common/globals.h

```diff
@@ -44,6 +44,15 @@
 
 
 /* ****************************************************************************
+ *
+ * MAX_ID_LEN
+ */
+#ifndef MAX_ID_LEN
+#define MAX_ID_LEN 256
```

Comment:
We have now (I think is was a recent addition) a limits.h file where I think this MAX_ID_LEN will fit better.